SETUP_LOCAL_CLUSTER.md

```diff
@@ -0,0 +1,70 @@
+# Introduction
+
+For both HOPR app and core protocol developers, being able to spin a fully interconnected HOPR network is critical.
+Without doing so, app developers will only be able to fake interactions with a HOPR node by mocking their API[^1], and
+core protocol developers will not be able to replicate critical functionality over messaging such as ticket redemption
+and packet relaying.
+
+Although the HOPR Association provides a series of [nodes](https://status.hoprnet.org/) anyone could talk in production[^2],
+this setup is not ideal for development as doing so would incur on costs for executing basic operations. Staging networks
+such as görli are too slow to provide meaningful feedback, so the ideal setup for properly interacting with the HOPR
+ecosystem toolset is running a local HOPR network, either in your workstation or via a local environment like
+[Gitpod](https://gitpod.io). This way contributors are not constrained by internet connection or blockchain RPC providers.
+
+# Tooling
+
+As of time of writing, the best way to setup a local HOPR network is by following these steps[^3]. Please bear in mind that these steps
+had been tested only on `macOS` and `Linux` devices, for Windows please use our cloud setup instead.
+
+1. **Download the latest version of the repository**: Download a local version of our [GitHub repository monorepo](https://github.com/hoprnet/hoprnet)[^4]
+   and unzip it in your local folder (roughly `~30 Mbs` at the time of writing). For the next tasks, we will assume you are within that folder.
+
+```
+wget https://github.com/hoprnet/hoprnet/archive/refs/heads/master.zip
+unzip master.zip
+cd hoprnet-master
+```
+
+2. **Install the dependencies of the project and build it**: Make sure you have `nodejs@16` (we suggest installing it via [nvm](https://github.com/nvm-sh/nvm), ie `nvm install lts/gallium`), and `yarn` (included in `nodejs@16` by running `corepack enable`).
+   to install and build the required packages and project modules. Please bear in mind that this process will take at least 5-10 minutes depending on your computer.
+
+```
+yarn && yarn build
```

Comment:
`yarn build` requires on Ubuntu the package `build-essentials` and on MacOS the XCode CLI Tools

Reply:
I spun up a brand new VM with `Ubuntu 20.04` and didn't install it. It might have been pre-installed as part of the installation process. Either way, I think it's worth adding just in case, let me add a footnote there.